Source/MillicastPublisher/Private/WebRTC/Texture2DVideoSourceAdapter.cpp

```diff
@@ -62,7 +62,36 @@ void FTexture2DVideoSourceAdapter::OnFrameReady(const FTexture2DRHIRef& FrameBuf
 	}
 
 	FPublisherStats::Get().FrameRendered();
+#else
+#if ENGINE_MAJOR_VERSION < 5 || ENGINE_MINOR_VERSION == 0
+	FRHIResourceCreateInfo CreateInfo(TEXT("VideoCapturerBackBuffer"));
+	FTexture2DRHIRef Texture = GDynamicRHI->RHICreateTexture2D(CaptureWidth, CaptureHeight, EPixelFormat::PF_B8G8R8A8, 1, 1, TexCreate_Shared | TexCreate_RenderTargetable, ERHIAccess::CopyDest, CreateInfo);
```

Comment:
CaptureWidth and CaptureHeight are not defined here, can you replace it with FrameBuffer->GetTexture2D()->GetSizeX() and FrameBuffer->GetTexture2D()->GetSizeY() ?